ci-operator/step-registry/ipi/conf/manual-creds/remove-unnecessary-creds/ipi-conf-manual-creds-remove-unnecessary-creds-commands.sh

```diff
@@ -37,6 +37,7 @@ echo "OCP Version: $ocp_version"
 
 v411="baremetal marketplace openshift-samples"
 v412="baremetal marketplace openshift-samples Console Insights Storage CSISnapshot"
+v413="baremetal marketplace openshift-samples Console Insights Storage CSISnapshot"
```

Comment:
How about we always set the default of the new version which equal to the last version's unless we get know the newly added operators.

Reply:
updated.